local-dev/k3d-seed-data/seed-user-keys.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+send_graphql_query() {
+  data=$(echo "mutation addKeyToUser {
+    addUserSSHPublicKey(input: {
+      name: \"${1}\"
+      publicKey: \"${2}\"
+      user: {
+        email: \"${1}@example.com\"
+      }
+    }) {
+      id
+      name
+    }
+  }" | sed 's/"/\\"/g' | sed 's/\\n/\\\\n/g' | awk -F'\n' '{if(NR == 1) {printf $$0} else {printf "\\n"$$0}}')
```

Comment:
```suggestion
  }" | sed 's/"/\\"/g' | sed 's/\\n/\\\\n/g' | awk -F'\n' '{if(NR == 1) {printf $0} else {printf "\\n"$0}}')
```
I was getting the error
```
Configuring ssh-key for guest@example.com
awk: illegal field $(mutation addKeyToUser {), name "0"
 input record number 1, file 
 source line number 1
{"errors":[{"message":"Syntax Error: Unexpected <EOF>.","locations":[{"line":3,"column":9}]}]}
Configuring ssh-key for reporter@example.com
```
which i fixed with the above suggestion